shoop_tests/admin/test_order_creator.py

```diff
@@ -279,11 +285,27 @@ def test_editing_existing_order(rf, admin_user):
 
     # Check that the product content is updated based on state
     assert edited_order.lines.count() == 5
-    assert edited_order.creator == admin_user
```

Comment:
Why this assert is removed? Shouldn't the creator still be admin_user? And that's important to check that it hasn't changed.